docs/SERVER.md

```diff
@@ -0,0 +1,311 @@
+
+- `Enable store log to restore queues and messages on server restart (Yn):`
+
+  Enter `y` to enable saving and restoring messages upon restarting the server.
+  
+- `Enable logging daily statistics (yN):`
+
+  Enter `y` to enable logging statistics for `Prometheus/Grafana`.
+  
+- `Require a password to create new messaging queues?`
+
+  Enter `r` or your arbitrary password to password-protect `smp-server`, or `n` to disable password protection.
+  
+- `Enter server FQDN or IP address for certificate (127.0.0.1):`
+
+  Enter your domain or ip address to serve `smp-server` from.
+
+### Manually
+
+Execute the following command:
+
+```sh
+sudo su smp -c "smp-server init -h"
+
+...
+Available options:
+  -l,--store-log           Enable store log for persistence
+  -s,--daily-stats         Enable logging daily server statistics
+  -a,--sign-algorithm ALG  Signature algorithm used for TLS certificates:
+                           ED25519, ED448 (default: ED448)
+  --ip IP                  Server IP address, used as Common Name for TLS online
+                           certificate if FQDN is not supplied
+                           (default: "127.0.0.1")
+  -n,--fqdn FQDN           Server FQDN used as Common Name for TLS online
+                           certificate
+  --no-password            Allow creating new queues without password
+  --password PASSWORD      Set password to create new messaging queues
+  -y,--yes                 Non-interactive initialization using command-line
+                           options
+  -h,--help                Show this help text
+```
+
+You should determine which flags are valid for your use-case and then execute `smp-server init` with `-y` flag for non-interactive initialization:
+
+```sh
+sudo su smp -c "smp-server init -y -<your flag> <your option>"
+```
+
+For example:
+
+```sh
+sudo su smp -c "smp-server init -y -l --ip 192.168.1.5 --password test"
+```
+
+... to initilize your `smp-server` configuration with `-l` flag to enable storing messages, `--ip` flag with `192.168.1.5` value to configure using ip and with `--password` flag with `test` value to password-protect `smp-server`.
+
+---
+
+After that, your installation is complete and you should see in your teminal output something like this:
+
+```sh
+Certificate request self-signature ok
+subject=CN = 127.0.0.1
+Server initialized, you can modify configuration in /etc/opt/simplex/smp-server.ini.
+Run `smp-server start` to start server.
+----------
+You should store CA private key securely and delete it from the server.
+If server TLS credential is compromised this key can be used to sign a new one, keeping the same server identity and established connections.
+CA private key location: /etc/opt/simplex/ca.key
+----------
+SMP server v3.4.0
+Fingerprint: d5fcsc7hhtPpexYUbI2XPxDbyU2d3WsVmROimcL90ss=
+Server address: smp://d5fcsc7hhtPpexYUbI2XPxDbyU2d3WsVmROimcL90ss=:V8ONoJ6ICwnrZnTC_QuSHfCEYq53uLaJKQ_oIC6-ve8=@<hostnames>
+```
+
```

Comment:
```suggestion

The server address above should be used in your client configuration and if you added server password it should only be shared with the other people when you want to allow them to use your server to receive the messages (all your contacts will be able to send messages, as it does not require a password). If you passed IP address or hostnames during the initialisation, they will be printed as part of server address, otherwise replace `<hostnames>` with the actual server addresses.
```